docs/address_lookup/a_brief_history_of_postcodes.md

```diff
@@ -0,0 +1,13 @@
+# A brief history of postcodes
+
+Postcodes in the UK date back to 1857 and originated in London.  A project team, led by team Sir Rowland Hill, set out to solve a problem that was identified the previous year where 100 million items of post were sent to a population of around 3 million over an area with a radius of ~12 miles centred around the St Paul's Cathedral post office.  The postal service was struggling!
+
+![Sir Rowland Hill](images/IMG_0205.jpg)
+
+<p><center><strong>Sir Rowland Hill</strong></center></p>
+
+To solve this problem the team divided the London into 10 districts - 1 district per standard compass point (N, NE, E, SE, S, SW, W, NW), and 2 districts for the centre of London - East Central (EC) and West Central (WC).  South London was further split into South East (SE) and South West (SW) London in 1868.
```

Comment:
You mention SE and SW twice. Is slightly confusing, were all 10 points being used from the start?

Reply:
Good spot thanks Jon!  They created 10 initially - then the South district was further split into 2, so sort of South-SE and South-SW.  I'll try to clarify.